docs/ref/settings.txt

```diff
@@ -90,14 +90,20 @@ If the ``Host`` header (or ``X-Forwarded-Host`` if
 list, the :meth:`django.http.HttpRequest.get_host()` method will raise
 :exc:`~django.core.exceptions.SuspiciousOperation`.
 
-When :setting:`DEBUG` is ``True`` or when running tests, host validation is
-disabled; any host will be accepted. Thus it's usually only necessary to set it
-in production.
+When :setting:`DEBUG` is ``True``, host validation is disabled; any host will
+be accepted. Thus it's usually only necessary to set it in production.
 
 This validation only applies via :meth:`~django.http.HttpRequest.get_host()`;
 if your code accesses the ``Host`` header directly from ``request.META`` you
 are bypassing this security protection.
 
+.. versionchanged:: 1.10
+
+    Beginning in Django 1.10, ``ALLOWED_HOSTS`` is also checked when running
```

Comment:
Make this note "self-contained" as described at https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.